### DISCUSSION TITLE
Looking for support about eslintrc format and future migrations

### DISCUSSION THREAD

**Discussion by Polm1** [nrwl/nx#6261] [General]
I’m looking for support about `eslintrc` format: currently in my company monorepo we are using `.json`  but I’d like to switch to `.js ` format because I need to use [this plugin](https://github.com/godaddy/eslint-plugin-i18n-json) to lint translations files and will be better to take advantage of `.js` format (need to implement an exported function).

Switching to `.js` could be an issue with future migrations of Nx updates and/or there can be other pitfalls?

Thanks in advance

**Comment by meeroslav** [comment]
Hi @Polm1,

Thank you for your question. It's hard to say what could happen in the future. Our migrations (and NX) at the moment assume `eslintrc` is in json format, so there might be breaks during the migration.

I invite you to open an issue for NX to handle `.eslintrc.json` and `.eslintrc.js` as equal citizens. Please state the scenario you have and we'll do our best to implement it as fast as possible.

Additionally, you are more than welcome to implement it yourself. We are always happy to get community PRs.

**Reply by Polm1** [reply]
Hi @meeroslav ,

Many thanks for your prompt reply: I will open an issue as you suggested.

I'd like to contribute more actively but I'm still on a early confidence phase with NX, so maybe it's a little bit too early to get deep into this, but thanks anyway for suggestion.

**Reply by tmtron** [reply]
> Hi @meeroslav ,
> 
> Many thanks for your prompt reply: I will open an issue as you suggested.
> 
> I'd like to contribute more actively but I'm still on a early confidence phase with NX, so maybe it's a little bit too early to get deep into this, but thanks anyway for suggestion.

Can you please post a link to the issue that you have created?

**Reply by meeroslav** [reply]
> > Hi @meeroslav ,
> > Many thanks for your prompt reply: I will open an issue as you suggested.
> > I'd like to contribute more actively but I'm still on a early confidence phase with NX, so maybe it's a little bit too early to get deep into this, but thanks anyway for suggestion.
> 
> Can you please post a link to the issue that you have created?

Here is the link to the original issue @Polm1 created: https://github.com/nrwl/nx/issues/6330

**Reply by Polm1** [reply]
Sorry, guys: been out for a couple of days and didn't read those updates.

Hope everything is fine now, thanks @meeroslav for reply
